/arena_camera_node:
  ros__parameters:
    camera_info_url: file:///home/volt/projects/volt_drivers_ws/src/arena_camera/config/camera_front_half.yaml
    camera_name: camera_1
    exposure_auto: true
    exposure_target: 34797
    fps: 15
    frame_id: camera_top/camera_link
    gain_auto: true
    gain_target: 1
    gamma_target: 1.0
    horizontal_binning: 4
    pixel_format: bgr8
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    resize_image: false
    serial_no: 220300119
    use_sim_time: false
    vertical_binning: 4
